config/checkstyle/checkstyle.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'checkstyle'
+
+def configDir = "${project.rootDir}/config/checkstyle"
+def reportsDir = "${project.buildDir}/reports"
+
+// For latest version, check: https://checkstyle.sourceforge.io/releasenotes.html
+checkstyle.toolVersion = '8.26'
+
+tasks.register("checkstyle", Checkstyle) {
```

Comment:
Please add file comment stating that this is used by Cloud Build and other Gradle builds (IDE, command-line).